src/ctia/stores/es/crud.clj

```diff
@@ -179,7 +178,8 @@
                                              (make-es-read-params params))
                          :_source
                          coerce!)]
-        (if (allow-read? doc ident)
+        (if (or (not (restricted-read? ident))
+                (allow-read? doc ident))
```

Comment:
Could you add some tests related to `:authorized-anonymous` field in the identity?